Automattic/woocommerce-payments
# Add "update business details" task to the "things to do next" list.

Fixes #1661.

#### Changes proposed in this Pull Request



<img width="558" alt="Screen Shot 2021-06-03 at 10 22 41 AM" src="https://user-images.githubusercontent.com/63922/120661366-6dde3a00-c445-11eb-8ecc-dd6c07a277db.png">

This PR adds any outstanding setup tasks to the WooCommerce Home screen task list(s) using the `woocommerce_admin_onboarding_task_list` filter. Currently, the "update business details" task is the only one implemented, but this change will filter any additional tasks into the WC home screen.

Additionally, the descriptive text for the task was changed to use the `additionalInfo` property instead of `content` since `content` is no longer visible by default (as of https://github.com/woocommerce/woocommerce-admin/pull/7089).







#### Testing instructions





* Use the testing instructions on https://github.com/Automattic/woocommerce-payments/pull/1799
* Verify that when the task is visible on the Payments > Overview, it is also visible on WooCommerce > Home

-------------------
